requirements.txt

```diff
@@ -8,7 +8,6 @@ numpy==1.14.5
 typing==3.6.4
 ruamel.yaml==0.15.66
 requests==2.20.0
-keras==2.1.6
```

Comment:
also needs to be removed from `setup.py`

Reply:
done